package.json

```diff
@@ -48,8 +48,8 @@
     "eslint-plugin-react": "^6.10.3",
     "jest": "^19.0.2",
     "prettier": "^1.2.2",
-    "react": "^15.5.0",
-    "react-dom": "^15.5.0"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
   },
```

Comment:
Can we add react and react-dom peerDependencies of `0.14.x || ^15.0.0 || ^16.0.0`?

Reply:
Done